test/copy_zipped.yml

```diff
@@ -0,0 +1,45 @@
+---
+# Tests for copy_zipped strategy
+- name: Given no previous deploy with download strategy
+  hosts: all
+  vars:
+    ansistrano_deploy_to: "/tmp/copy_zipped/my-app.com"
+  tasks:
+    - name: Assert ansistrano_deploy_to path does not exist
+      stat:
+        path: "{{ ansistrano_deploy_to }}"
+      register: st
+    - debug:
+        msg: "Path does not exist and is a directory"
+      when: st.stat.exists is defined and not st.stat.exists
+
+- name: When deploying using copy_zipped strategy
+  hosts: all
+  vars:
+    ansistrano_deploy_via: "copy_zipped"
+    ansistrano_deploy_from: /var/log
+    ansistrano_deploy_to: "/tmp/copy_zipped/my-app.com"
+  roles:
+    - { role: local-ansistrano }
+
+- name: Then a successful deploy with copy_zipped strategy should be done
+  hosts: all
+  vars:
+    ansistrano_deploy_to: "/tmp/copy_zipped/my-app.com"
+  tasks:
+    - name: Assert ansistrano_deploy_to path does exist
+      stat:
+        path: "{{ ansistrano_deploy_to }}"
+      register: st
+    - debug:
+        msg: "Path exists and is a directory"
+      when: st.stat.exists is defined and st.stat.exists
+
+- name: And I should be able to do a second deploy
+  hosts: all
+  vars:
+    ansistrano_deploy_via: "download_unarchive"
```

Comment:
Change this to copy_zipped and the tests should work :)

Reply:
Oh... very thanks. :-)